airbyte-integrations/connectors/source-clockify/unit_tests/test_source.py

```diff
@@ -11,7 +11,7 @@
 def setup_responses():
     responses.add(
         responses.GET,
-        "https://api.clockify.me/api/v1/workspaces/workspace_id/users",
```

Comment:
What's the reason for changing the unit test here?

Reply:
I changed the base url for the tests in order to ensure that the Clockify API url is not hard coded